maud/getting_stan_inputs.py

```diff
@@ -66,17 +66,20 @@ def get_stan_inputs(
 
 def get_prior_inputs(parameters: ParameterSet) -> Tuple[Dict, Dict]:
```

Comment:
All this `get_*` could receive a train argument and act accordingly, return only one `dict`. It is fine for now, though.

Reply:
Yeah agreed, I think there's plenty of scope for further improvements generally.